src/main/java/java8/Student.java

```diff
@@ -0,0 +1,30 @@
+package java8;
+
+public class Student {
```

Comment:
* Please use lombok API annotations to get rid of boilerplates code like getters, setters, and constructors.
https://www.baeldung.com/intro-to-project-lombok